reference/functions/int.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: int
+published: true
+tags: [reference, functions, int]
+---
+
+[%CFEngine_function_prototype(string)%]
+
+**Description:** Convert numeric string to int.
+
+[%CFEngine_function_attributes(string)%]
+
+If `string` represents a floating point number then the decimals are *truncated*.
+
+```cfengine3
+bundle agent main
+{
+  vars:
+      "real_str_1" string => "123.456";
+      "real_str_2" string => "567.789";
+
+      "int1" int => int("${real_str_1}");
+      "int2" int => int("${real_str_2}");
+
+  reports:
+      '"${real_str_1}" -> ${int1}';
+      '"${real_str_2}" -> ${int2}';
+}
+```
+
+Output:
+
+```
+R: "123.456" -> 123
+R: "567.789" -> 567
+```
+
+**Example:**
+
+[%CFEngine_include_example(int.cf)%]
```

Comment:
Oh, you have an example here, so I think you should delete the embedded example and output.